prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -610,4 +610,4 @@ periodics:
+      - "-images=projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-1604-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-1804-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-1804-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2004-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2004-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2204-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2204-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2210-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2210-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2304-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2304-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2310-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2310-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2404-lts-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2404-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-1604-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-1804-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-1804-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2004-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2004-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2204-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2204-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2210-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2210-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2304-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2304-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2310-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2310-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2404-lts-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2404-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2410-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2410-arm64,projects/bct-prod-images/global/images/family/windows-2025,projects/bct-prod-images/global/images/family/windows-2025-core"
```

Comment:
The `ubuntu-daily-cit-shapevalidation` dashboard is intended for Canonical for their development builds of ubuntu only. We don't have a periodic dashboard for running shapevalidation against public cloud images right now.

If you copy and paste this to a new `cit-periodic-shapevalidation` dashboard with the non-ubuntu images that works, or a TODO is ok. Having an extra image here for now is not a big deal.